src/physics/site_types/qubit.jl

```diff
@@ -175,10 +175,10 @@ function ITensors.op(::OpName"Ry", ::SiteType"Qubit"; θ::Number)
 end
 
 # Rotation around Z-axis
-function ITensors.op(::OpName"Rz", ::SiteType"Qubit"; ϕ::Number)
+function ITensors.op(::OpName"Rz", ::SiteType"Qubit"; θ::Number)
```

Comment:
This would be breaking, though I think it would be nice to change it to be consistent with "Rx" and "Ry".

Maybe we can deprecate it quietly for now, say with:
```julia
ITensors.op(::OpName"Rz", ::SiteType"Qubit"; θ=nothing, ϕ=θ)
  isnothing(θ) && error("Must specify gate angle in Rz gate.")
  # [...]
end
```
so then people can use `θ` going forward, but it won't break if they are still using `ϕ` and don't specify `θ`.

Reply:
Made it a separate pass-through so people don't try to pass `theta` and `phi` simultaneously

Reply:
The way you wrote it, one method will overwrite the other since the arguments have the same signatures (and kwargs can't be used for dispatch).

We could check for that explicitly:
```julia
ITensors.op(::OpName"Rz", ::SiteType"Qubit"; θ=nothing, ϕ=nothing)
  isone(count(isnothing, (θ, ϕ))) || error("Must specify the keyword argument `θ` (or the deprecated `ϕ`) when creating an Rz gate, but not both.")
  # [...]
end
```

Reply:
> The way you wrote it, one method will overwrite the other since the arguments have the same signatures (and kwargs can't be used for dispatch).

lol forgot about this one. i love kwargs.